### ISSUE TITLE
Please make the APK available again

### ISSUE THREAD

**Issue by IzzySoft** [PIVX-Project/PIVX-Android#13]
Could you please make the APK available again – ideally on the `releases/` page?

**Comment by IzzySoft** [comment]
Any chance?

**Comment by IzzySoft** [comment]
@furszy ^^? Or did you abandon the project meanwhile?

**Comment by IzzySoft** [comment]
No answer at all must certainly mean it's dead then. Sad.

**Comment by furszy** [comment]
Hi, sorry for the delay, have been busy with 3.2 core release and fake stake report.
 It will be updated after the release, come to discord if you want a quicker answer.

**Comment by IzzySoft** [comment]
Thanks @furszy! I just wanted to know if you'd make the APKs available again – and that you just confirmed :tada: Looking forward to it showing up then (a ping here would be appreciated once it's available again).

**Comment by furszy** [comment]
Yeah sure, will publish the new version that includes LNZP and the whole zerocoin wallet support upgrade.

Just finishing priorities first, it took some more time that what was planned to get to 3.2 but we are almost there.

**Comment by IzzySoft** [comment]
Then good "forthcoming"! Looking forward to your announcement.

**Comment by IzzySoft** [comment]
Just out of interest: any ETA yet?

**Comment by furszy** [comment]
All yours --> https://github.com/PIVX-Project/PIVX-Android/releases/tag/3.1.2

**Comment by IzzySoft** [comment]
Thanks! And good timing: a week ago I had said it got too big (wow, it almost tripled in size: 13M → 32M; guess you now support multiple platforms – ah, bingo, good! That justifies the exception – now). Last weekend I was able to upgrade my hardware a little, and decided to lift the limit accordingly (to ~30M per app) :laughing:

Activated auto-update again (now fetching from `releases/` – perfect place, double-thanks!) and fetched the current one (to verify the updater gets it). Will show up [in my repo](https://apt.izzysoft.de/fdroid/index/apk/pivx.tech.furszy) with the next sync tomorrow (and the note on the update delay removed).

If you wish for some screenshots, I happily oblige if you could provide some non-framed ones. Considering the small screen estate on mobile devices, framing wastes too much space and leaves the important details to pure guessing.

**Comment by IzzySoft** [comment]
PS: Ouch, it's ~100k too big to pass the VirusTotal API (they have an upload limit of 32M for their API) :cry: I've applied for "elevated privileges", but I've no idea how long that will take and whether I will get them granted…

Wondering if it could be squeezed a bit? But I only found peanuts which might be obsolete, like `assets/old/checkpoints` or `lib/x86_64/darwin/libscrypt.dylib` or the `META-INF/*.version` files. Doesn't sum up to enough "squeeze"…

**Comment by IzzySoft** [comment]
Just wondering where 3.1.3 and 3.1.4 went to – they are again missing. Do you no longer wish to make releases available?
